### PR TITLE
Add the min and max price properties on Add Business Form and servlets

### PR DESCRIPTION
**Pull request by dalisc**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/BusinessDataServlet.java [hunk 1]
```diff
@@ -94,19 +96,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+        addressLng = 404;
+    } else {
+        addressLng = Float.parseFloat(addressLngStr);
+    }
+    String contactDetails = request.getParameter(BUSINESS_CONTACT_INFO);
+    String orderDetails = request.getParameter(BUSINESS_ORDER_INFO);
+    String minPriceStr = request.getParameter(BUSINESS_MIN_PRICE);
+    String maxPriceStr = request.getParameter(BUSINESS_MAX_PRICE);
+    float minPrice;
+      if (minPriceStr.isEmpty()) {
```
Comment (jacoblipech) [inline src/main/java/com/google/sps/servlets/BusinessDataServlet.java:115]: Indentation is off here

File: src/main/java/com/google/sps/servlets/BusinessDataServlet.java [hunk 2]
```diff
@@ -94,19 +96,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String addressLatStr = request.getParameter(BUSINESS_ADDRESS_LAT);
     String addressLngStr = request.getParameter(BUSINESS_ADDRESS_LNG);
     float addressLat;
-      if (addressLatStr.isEmpty()) {
-          addressLat = 404;
+    if (addressLatStr.isEmpty()) {
+        addressLat = 404;
```
Comment (jacoblipech) [inline src/main/java/com/google/sps/servlets/BusinessDataServlet.java:100]: can consider changing 404 to a constant like `NOT_FOUND`?
Actually, I am not sure if 404 is a good implementation? cos for other parameters, frontend will check for `isEmpty()` but for float values, we use 404 for validation?

File: src/main/java/com/google/sps/servlets/BusinessDataServlet.java [hunk 3]
```diff
@@ -94,19 +96,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+          minPrice = 404;
       } else {
-          addressLat = Float.parseFloat(addressLatStr);
+          minPrice = Float.parseFloat(minPriceStr);
       }
-      float addressLng;
-      if (addressLngStr.isEmpty()) {
-          addressLng = 404;
+      float maxPrice;
+      if (maxPriceStr.isEmpty()) {
```
Comment (jacoblipech) [inline src/main/java/com/google/sps/servlets/BusinessDataServlet.java:121]: If the default value is all 404, can consider having a function which returns 404 when empty so its neater.

Comment (dalisc) [reply src/main/java/com/google/sps/servlets/BusinessDataServlet.java:121]: Sounds good! I'll implement this and submit a pull request

Comment (dalisc) [reply src/main/java/com/google/sps/servlets/BusinessDataServlet.java:121]: i mean merge this in

**Pull request merged**